### DISCUSSION TITLE
FPS drop significantly after yesterday's update

### DISCUSSION THREAD

**Discussion by shanenicholson88** [flybywiresim/aircraft#3927] [Help] [answered]
Good evening all, thank you for the great work to start.
My FS settings have not changed but my FPS when flying the sim have dropped at least 30%.. Any ideas?

**Comment by Paytheon** [comment, marked as answer]
This is mostly a MSFS 2020 Release 3 issue according to the bugs reported:
1. https://forums.flightsimulator.com/t/severe-performance-problem-in-1-14-5-0-identified-and-reported/376863
2. https://forums.flightsimulator.com/t/stutter-and-stop-at-10fps-introduced-with-worldupdate3-still-there-with-1-14-5-0/376094
